sema/subyt/sources.py

```
@@ -90,17 +93,63 @@ def mime_from_identifier(identifier: str) -> str:
+    def _parse_source_identifier(identifier: str | Path | dict) -> dict:
+        if isinstance(identifier, dict):
+            config = identifier
+            if "identifier" not in config:
+                config["identifier"] = str(config["path"])
+                config["path"] = Path(config["path"])
+            return config
+        # else
+        if isinstance(identifier, Path):
+            return {"identifier": identifier, "path": identifier}
+        # else
+        config: dict = dict()
```

Ruff (0.8.2)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)